prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public class LicenseDetailConfiguration : SeededTable<LicenseDetail>
         public static readonly DateTime June1st2022Cohort = new(2022, 6, 1, 8, 0, 0, DateTimeKind.Utc);
         public static readonly DateTime July15th2022 = new(2022, 7, 15, 8, 0, 0, DateTimeKind.Utc);
         public static readonly DateTime Aug1st2022 = new(2022, 8, 1, 8, 0, 0, DateTimeKind.Utc);
+        public static readonly DateTime Aug2nd2022 = new(2022, 8, 2, 8, 0, 0, DateTimeKind.Utc);
```

Comment:
I think the name should be more meaningful, like `EnableNonPrescriberPrefixAug2nd2022`